extensions/attention-stream/src/background.ts

```diff
@@ -59,13 +59,11 @@ const enableDevMode = Boolean(__ENABLE_DEVELOPER_MODE__);
 // eslint-disable-next-line no-undef
 const enableEmulatorMode = Boolean(__ENABLE_EMULATOR_MODE__);
 
-const GLEAN_MAX_URL_LENGTH = 2048;
-
 // The Rally-assigned Study ID.
 let studyId = "attentionStream";
 
 // The "slug" on the extension store.
-let storeId = "rally-attention-stream";
```

Comment:
I'm not 100% sure what the rules on the CWS are about the URL - it seems to be derived from the name so I think this will change from this: https://chrome.google.com/webstore/detail/rally-attention-stream/bahhehaddofgkccippmjcecepdakppme to this: https://chrome.google.com/webstore/detail/mozilla-rally/bahhehaddofgkccippmjcecepdakppme, but if I'm wrong then we'll need to figure out per-browser store IDs.

Reply:
can you think of a way to test this before it goes live?

Reply:
I guess worst case I'll just change it to `mozilla-rally` for now; and if the Chrome URL doesnt change we can push a quick fix for per-browser. The Chrome store seems to accept things much faster so hopefully that's not an issue for too long. It just breaks attribution right?